### PR TITLE
Migrate Elder Kuruti to SAI

### PR DESCRIPTION
**Pull request by velinath**
**Changes proposed**:

- Another SAI migration.

**Target branch(es)**: 335

### CODE REVIEW

**Comment by tkrokli** [comment]
OK, I see you have found out that `gossip_menu` and a couple of other tables have changed column/field names.

They were changed by ChazyTheBest at 2017-05-04:
- See commit https://github.com/TrinityCore/TrinityCore/commit/5ff8601d36a14e95f9c227db1b85253ff36e551e `sql/updates/world/3.3.5/2017_05_04_02_world.sql` for `gossip_menu` and `gossip_menu_option`
- See commit https://github.com/TrinityCore/TrinityCore/commit/d17a45ee7e348d6fc9146dd81e0457316eef1a62 `sql/updates/world/3.3.5/2017_05_04_00_world.sql`) for `creature_text`

File: sql/updates/world/3.3.5/2017_09_25_99_world.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-- Add gossip text to menu
+
+DELETE FROM `gossip_menu` WHERE `entry` BETWEEN 57020 AND 57022;
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `VerifiedBuild`) VALUES
```
Comment (tkrokli) [inline sql/updates/world/3.3.5/2017_09_25_99_world.sql]:
```
MenuID  TextID  VerifiedBuild
------  ------  -------------
```

**Comment by tkrokli** [comment]
Heh. Minor side note: I have become so used to the TC tracker labels that I find the default ones unusable... :-p

**Comment by velinath** [comment]
are there still column headers? i'm staring at this sql, maybe i'm just missing them

**Comment by tkrokli** [comment]
Maybe I used the wrong terminology. I meant the field names, like `ID`, `entry` etc.

**Comment by velinath** [comment]
Yeah, I just am not seeing what I missed

**Comment by tkrokli** [comment]
BTW, I just posted an invite for you to be a collaborator on my repository, in case you would like to make adjustments. 😸

File: sql/updates/world/3.3.5/2017_09_25_99_world.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
+-- Add gossip text to menu
+
+DELETE FROM `gossip_menu` WHERE `MenuID` BETWEEN 57020 AND 57022;
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
+(57020,9227,0),(57021,9229,0),(57022,9231,0);
+
+DELETE FROM `gossip_menu_option` WHERE `MenuID` BETWEEN 57020 AND 57022 OR `MenuID` = 7582;
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`)
```
Comment (tkrokli) [inline sql/updates/world/3.3.5/2017_09_25_99_world.sql]: missing keyword `VALUES` at the end of this line.

**Comment by velinath** [comment]
issue: currently Kuruti does not give the player the item. there does not appear to be a spell, but the ADD_ITEM smart action doesn't seem to work. tried switching the gossip menu change to a linked action but no difference.

**Comment by velinath** [comment]
spell worked. just gotta revert my menu crap and make sure it still works then

**Pull request closed without merging**